### DISCUSSION TITLE
Log "channel sent a message from native to Flutter on a non-platform thread"  Can crash the windows app?

### DISCUSSION THREAD

**Discussion by ciceromsjr** [firebase/flutterfire#12971] [Q&A]
I am seeing the log when starting my app on my computer. Although  it says that it may cause crashes, it works. But some users are experiencing crash when starting the app on Windows.
I wonder if it is related to this log:

`[ERROR:flutter/shell/common/shell.cc(1055)] The 'plugins.flutter.io/firebase_firestore/document/d060c6b4-02d2-4cfd-9884-fe19303646ad' channel sent a message from native to Flutter on a non-platform thread. Platform channel messages must be sent on the platform thread. Failure to do so may result in data loss or crashes, and must be fixed in the plugin or application code creating that channel.
See https://docs.flutter.dev/platform-integration/platform-channels#channels-and-platform-threading for more information.
[MF] Done!
[ERROR:flutter/shell/common/shell.cc(1055)] The 'plugins.flutter.io/firebase_firestore/query/534d1ac2-50ac-4528-ae4b-fbf95238324e' channel sent a message from native to Flutter on a non-platform thread. Platform channel messages must be sent on the platform thread. Failure to do so may result in data loss or crashes, and must be fixed in the plugin or application code creating that channel.
See https://docs.flutter.dev/platform-integration/platform-channels#channels-and-platform-threading for more information.`

**Comment by shtse8** [comment]
I have the same issue on window

**Comment by CoderGangW** [comment]
Same here, I happen when I use the audioplayer package. Did you solve it by any chance?

**Reply by jonahzheng** [reply]
> Same here, I happen when I use the audioplayer package. Did you solve it by any chance?

the same isues in windows
